### DISCUSSION TITLE
Data Science Libraries

### DISCUSSION THREAD

**Discussion by metedogan** [wilfredinni/python-cheatsheet#233] [Ideas]
Hello,
I love the concept of your website and content. It helped to me so much.
I am learning Data Science right now and when I face a problem  I'm struggling to find resources about the Pandas, NumPy, Seaborn and  Sc-kit. It would be great if you include these libraries. (I apologize if I made some misspelling English is not my main language)

**Comment by wilfredinni** [comment]
Hi, I am glad you like the project!

>  am learning Data Science right now and when I face a problem I'm struggling to find resources about the Pandas, NumPy, Seaborn and Sc-kit. It would be great if you include these libraries.

Currently, our focus will be limited to Python's built-in functionalities and the standard library. The field of data science is vast and complex enough to potentially justify a separate project.
